src/platform/user/tests/selectors.unit.spec.js

```diff
@@ -516,6 +516,33 @@ describe('user selectors', () => {
     });
   });
 
+  describe('isNotInMPI', () => {
+    it('returns `true` if the profile.status is `NOT_FOUND`', () => {
+      const state = {
+        user: {
+          profile: {
+            status: 'NOT_FOUND',
+          },
+        },
+      };
+      expect(selectors.isNotInMPI(state)).to.be.true;
+    });
+    it('returns `false` if the profile.status is anything other than `NOT_FOUND`', () => {
+      const state = {
+        user: {
+          profile: {
+            status: 'NOT_AUTHORIZED',
+          },
+        },
+      };
+      expect(selectors.isNotInMPI(state)).to.be.false;
+      delete state.user.profile.status;
+      expect(selectors.isNotInMPI(state)).to.be.false;
+      delete state.user.profile;
+      expect(selectors.isNotInMPI(state)).to.be.false;
```

Comment:
I'm not going to hold up the PR for this, but for the sake of clarity it might be good to break these tests up and be more specific with the test names. So, something like:
```
it('returns false if the profile.status is anything other than "NOT_FOUND")
it('returns false if the profile.status does not exist)
it('returns false if the profile does not exist')
```

Reply:
Thanks. Was just following a pattern that was used in this file already (which I guess I actually introduced six months ago 😆 ) and is mostly done as a safety check to make sure the selector doesn't blow up when trying to access a deeply-nested property that doesn't exist.